filebeat/input/filestream/input.go

```diff
@@ -85,28 +86,66 @@ func Plugin(log *logp.Logger, store statestore.States) input.Plugin {
 	}
 }
 
+func updateDefaults(cfg config, inactiveSet bool) config {
```

Comment:
Maybe name this something more specific to its purpose like `applyDeletionOverrides`